README.md

```diff
@@ -63,9 +62,7 @@ Otherwise simply follow the instructions for any django based deployment
 
 ## Updating
 0. Before updating it is recommended to **backup your database**
-1. Stop the container using `docker-compose down`. 
```

Comment:
there is no explicit `docker-compose down` needed. A `docker-compose up` recreated the container if necessary. For example the database would not be recreated and keeps running

Reply:
are you sure of this ? i had mixed experiences when running docker compose up again when the containers where already up and also restart sometimes posed problems since it appeared to not behave 100% the same as down and up. But maybe that was just a bug ...

Reply:
There may be issues if you change for example the container names. But otherwise there should be no problems. It recreates if necessary